node/cmd/ccq/http.go

```diff
@@ -112,17 +112,17 @@ func (s *httpServer) handleQuery(w http.ResponseWriter, r *http.Request) {
 		Signature:    signature,
 	}
 
-	requestId := hex.EncodeToString(signedQueryRequest.Signature)
-	s.logger.Info("received request from client", zap.String("userId", permEntry.userName), zap.String("requestId", requestId))
-
 	if status, err := validateRequest(s.logger, s.env, s.permissions, s.signerKey, apiKey, signedQueryRequest); err != nil {
-		s.logger.Error("failed to validate request", zap.String("userId", permEntry.userName), zap.String("requestId", requestId), zap.Int("status", status), zap.Error(err))
+		s.logger.Error("failed to validate request", zap.String("userId", permEntry.userName), zap.String("requestId", hex.EncodeToString(signedQueryRequest.Signature)), zap.Int("status", status), zap.Error(err))
 		http.Error(w, err.Error(), status)
 		// Error specific metric has already been pegged.
 		invalidRequestsByUser.WithLabelValues(permEntry.userName).Inc()
 		return
 	}
 
+	requestId := hex.EncodeToString(signedQueryRequest.Signature)
+	s.logger.Info("received request from client", zap.String("userId", permEntry.userName), zap.String("requestId", requestId))
```

Comment:
Don't know why you didn't just move the log message and leave the requestId there?  But, okay.

Reply:
That was actually the bug. For unsigned requests, the requestId is set in `validateRequest`.